Add `charon exit` to voluntary exits (#373)
ObolNetwork/obol-docs: docs/start/quickstart-exit.md

        </ul>
        <pre>
          <code>
            {String.raw`docker exec -it charon-distributed-validator-node-lodestar-1 /bin/sh -c 'node /usr/app/packages/cli/bin/lodestar validator voluntary-exit \
            --beaconNodes="http://charon:3600" \
            --dataDir=/opt/data \
            --exitEpoch=162304 \
            --network=goerli \
            --yes'`}
          </code>
        </pre>
      </TabItem>
      <TabItem value="charon" label="Charon" default>

>       Voluntary exit can be submitted directly through charon as well. The partially signed exit messages are stored (centrally) on Obol's infrastructure. Exits through charon are submitted per validator. All active validators' public keys can be listed with:

Reviewer: ```suggestion
      Voluntary exit can be submitted directly through Charon as well. The partially signed exit messages are stored (centrally) on Obol's infrastructure. Exits through Charon are submitted per validator. All active validators public keys for a given cluster lock can be listed with:
```

        <pre>
          <code>

>         {String.raw`docker exec -it charon-distributed-validator-node-charon-1 /bin/sh -c 'charon exit active-validator-list'`}

Reviewer: I think you need to pass a BN flag here.

          </code>
        </pre>
      Then a signed partial exit for validator can be submitted by:
        <pre>
          <code>
        {String.raw`docker exec -it charon-distributed-validator-node-charon-1 /bin/sh -c 'charon exit sign \
    --beacon-node-endpoints="http://lighthouse:5052" \
    --validator-public-key="<VALIDATOR_PUBLIC_KEY>" \
    --publish-timeout="5m"'`}
          </code>
        </pre>

>       After the threshold of signed partial exits is met, any operator from the cluster can broadcast the full signed exit to the beacon chain:

Reviewer: While technically correct, we might want to be a little more explicit with "threshold of signed partial exits is met. A user doesn't know that threshold happens when threshold operators broadcasted their partial exit :D

            --beaconNodes="http://charon:3600" \
            --dataDir=/opt/data \
            --exitEpoch=162304 \
            --network=goerli \
            --yes'`}
          </code>
        </pre>
      </TabItem>
      <TabItem value="charon" label="Charon" default>
      Voluntary exit can be submitted directly through Charon as well. The partially signed exit messages are stored (centrally) on Obol's infrastructure. Exits through Charon are submitted per validator. All active validators public keys for a given cluster lock can be listed with:
        <pre>
          <code>
        {String.raw`docker exec -it charon-distributed-validator-node-charon-1 /bin/sh -c 'charon exit active-validator-list \
    --beacon-node-endpoints="http://lighthouse:5052"'`}
          </code>
        </pre>
      Then a signed partial exit for validator can be submitted by:
        <pre>
          <code>
        {String.raw`docker exec -it charon-distributed-validator-node-charon-1 /bin/sh -c 'charon exit sign \
    --beacon-node-endpoints="http://lighthouse:5052" \
    --validator-public-key="<VALIDATOR_PUBLIC_KEY>" \
    --publish-timeout="5m"'`}
          </code>
        </pre>

>       After a sufficient amount of signed partial exits from node operators in the cluster is cumulated, a full (complete) exit is created. The threshold is the same as the one submitted during creating of the cluster. After a full exit message is created, any operator from the cluster can broadcast it to the beacon chain with:

Reviewer: ```suggestion
      After a sufficient amount of signed partial exits from node operators in the cluster is cumulated, a full (complete) exit is created. The threshold is the same as the one submitted during the cluster creation. After a full exit message is created, any operator from the cluster can broadcast it to the beacon chain with:
```

            --beaconNodes="http://charon:3600" \
            --dataDir=/opt/data \
            --exitEpoch=256 \
            --network=holesky \
            --yes'`}
          </code>
        </pre>
      </TabItem>
      <TabItem value="charon" label="Charon" default>
      Voluntary exit can be submitted directly through charon as well. The partially signed exit messages are stored (centrally) on Obol's infrastructure. Exits through charon are submitted per validator. All active validators' public keys can be listed with:
        <pre>
          <code>
        {String.raw`docker exec -it charon-distributed-validator-node-charon-1 /bin/sh -c 'charon exit active-validator-list \
    --beacon-node-endpoints="http://lighthouse:5052"'`}
          </code>
        </pre>
      Then a signed partial exit for validator can be submitted by:
        <pre>
          <code>
        {String.raw`docker exec -it charon-distributed-validator-node-charon-1 /bin/sh -c 'charon exit sign \
    --beacon-node-endpoints="http://lighthouse:5052" \
    --validator-public-key="<VALIDATOR_PUBLIC_KEY>" \
    --publish-timeout="5m"'`}
          </code>
        </pre>

>       After a sufficient amount of signed partial exits from node operators in the cluster is cumulated, a full (complete) exit is created. The threshold is the same as the one submitted during creating of the cluster. After a full exit message is created, any operator from the cluster can broadcast it to the beacon chain with:

Reviewer: ```suggestion
      After a sufficient amount of signed partial exits from node operators in the cluster is cumulated, a full (complete) exit is created. The threshold is the same as the one submitted during the cluster creation. After a full exit message is created, any operator from the cluster can broadcast it to the beacon chain with:
```

            --beaconNodes="http://charon:3600" \
            --dataDir=/opt/data \
            --exitEpoch=194048 \
            --network=mainnet \
            --yes'`}
          </code>
        </pre>
      </TabItem>
      <TabItem value="charon" label="Charon" default>
      Voluntary exit can be submitted directly through charon as well. The partially signed exit messages are stored (centrally) on Obol's infrastructure. Exits through charon are submitted per validator. All active validators' public keys can be listed with:
        <pre>
          <code>
        {String.raw`docker exec -it charon-distributed-validator-node-charon-1 /bin/sh -c 'charon exit active-validator-list \
    --beacon-node-endpoints="http://lighthouse:5052"'`}
          </code>
        </pre>
      Then a signed partial exit for validator can be submitted by:
        <pre>
          <code>
        {String.raw`docker exec -it charon-distributed-validator-node-charon-1 /bin/sh -c 'charon exit sign \
    --beacon-node-endpoints="http://lighthouse:5052" \
    --validator-public-key="<VALIDATOR_PUBLIC_KEY>" \
    --publish-timeout="5m"'`}
          </code>
        </pre>

>       After a sufficient amount of signed partial exits from node operators in the cluster is cumulated, a full (complete) exit is created. The threshold is the same as the one submitted during creating of the cluster. After a full exit message is created, any operator from the cluster can broadcast it to the beacon chain with:

Reviewer: ```suggestion
      After a sufficient amount of signed partial exits from node operators in the cluster is cumulated, a full (complete) exit is created. The threshold is the same as the one submitted during the cluster creation. After a full exit message is created, any operator from the cluster can broadcast it to the beacon chain with:
```
